extension/chrome/dev/ci_pgp_host_page.htm

```diff
@@ -0,0 +1,26 @@
+<!-- © 2016-2018 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com -->
+
+<html>
+
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+  <title>FlowCrypt</title>
+  <link rel="stylesheet" href="/css/cryptup.css" />
+  <link rel="stylesheet" href="/css/sweetalert2.css" />
+  <link rel="icon" href="/img/logo/flowcrypt-logo-19-19.png" />
+</head>
+
+<body id="settings">
+  <pre style="text-align: left; font-family: monospace;"></pre>
+
+  <script src="/lib/purify.min.js"></script>
+  <script src="/lib/jquery.min.js"></script>
+  <script src="/lib/sweetalert2.js"></script>
+  <script src="/lib/openpgp.js"></script>
+  <script src="ci_pgp_host_page.js" type="module"></script>
+
+  <iframe src="chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/chrome/elements/pgp_block.htm?account_email=flowcrypt.compatibility%40gmail.com&frame_id=frame_yVMKFLRDiY&message=&has_password=___cu_false___&message_id=162275c819bcbf9b&sender_email=human%40flowcrypt.com&is_outgoing=___cu_false___&parentTabId=29%3A0" frameborder="0"></iframe>
```

Comment:
The problem is that `parentTabId` is a dynamic value - we need to figure out what it is before adding it to the url. I know you logged it in ci_pgp_host_page, but there is a chance that it has changed as you were testing it.

Actually, now that I'm looking at it, the tested frame does not _actually_ need to be an embedded frame. That will simplify things.

To (probably) fix this:
1) at the end of `ci_pgp_host_page.ts` do something like `$('body').append('<div data-test="tab-id">${Xss.escape(tabId)}</div>')`;
2) the test runner needs to retrieve the value with something like `hostPage.waitAll('@tab-id')` and `hostPage.read('@tab-id')`;
3) whenever opening a pgp_block during tests, it currently usually includes `parentTabId=0` or something like that. These should be removed, and instead you should add the dynamic value of whatever the host tabId actually is.
4) Then you can open the pgp_block as a separate tab just like we do today. It will be easier that way. It will work as long as it has the correct parentTabId and the "parent" tab remains open.